### PR TITLE
Search select multiple

### PR DESCRIPTION
**Pull request by el-fuego**
Add multi-selection mode for search-select. Based on default multiple select tag attribute

### CODE REVIEW

**Comment by ldlsegovia** [comment]
@el-fuego would you add the test to merge this PR? :)
https://github.com/platanus/activeadmin_addons/blob/master/spec/features/select_2_spec.rb
Thanks!

File: app/assets/javascripts/activeadmin_addons/select2.js [hunk 1]
```diff
@@ -98,6 +99,7 @@ $(function() {
 
       var select2Config = {
         width: '80%',
+        multiple: multiple,
```
Comment (mfkp) [inline app/assets/javascripts/activeadmin_addons/select2.js]: This should be opt-in, not default behavior.

**Comment by jalberto** [comment]
@el-fuego @ldlsegovia I tried this PR and I see 2 issues:

1. when you back to edit an object with a previosu relation, you can see a blank item in the selector.
2. the POST/PATCH od an string instead of an array (expecting: category_ids: [1,2,3] got: category_ids: ["1,2,3"]

**Comment by ldlsegovia** [comment]
https://github.com/platanus/activeadmin_addons/blob/master/docs/select2_tags.md control was made to achieve this goal.

**Pull request closed without merging**